src/components/Content/Content.tsx

```diff
@@ -1,143 +1,150 @@
-		contentForDisplay = show ? content : content.slice(0, 3);
-	}
+		const leftColumn = sectionsTranspose[0]?.flat(1);
+		const rightColumn = sectionsTranspose[1]?.flat(1);
 
-	/**
-	 * Renders the content of type "labelValue"
-	 * @param contentItem
-	 * @returns The content with a sctructure of label: value
-	 */
-	const labelValueContent = (contentItem: ContentType) => (
-		<LabelValueWrapper>
-			<Label>{contentItem.label}:</Label>
-			<Value>{contentItem.value}</Value>
-		</LabelValueWrapper>
-	);
+		return [leftColumn, rightColumn];
+	}, [sections, fieldDef]);
 
 	/**
-	 * Renders the content of type "paragraph"
-	 * @param contentItem
-	 * @returns A paragraph with the value of the content
+	 * Toggles the state use to show or hide the content.
 	 */
-	const paragraphContent = (contentItem: ContentType) => (
-		<LabelValueWrapper>
-			<Label>{contentItem.label}:</Label>
-			<Paragraph>{contentItem.value}</Paragraph>
-		</LabelValueWrapper>
-	);
+	const showDetails = () => {
+		setShowMore(!showMore);
+	};
 
 	/**
-	 * Renders the content of type "tags"
-	 * @param contentItem
-	 * @returns The a lists of tags/chips.
+	 * Given a column containing the positioned fields, checks if the
+	 * field exists and executes its transform function.
+	 * @param column that contains the names of the fields to render
+	 * @returns the JSX element created by the transform function.
 	 */
-	const tagsContent = (contentItem: ContentType) => {
-		if (!Array.isArray(contentItem.value)) {
-			return;
-		}
+	const renderColumn = (fields: string[], column: string) => {
+		return fields?.map((field) => {
+			const currentField = fieldDef?.find((fieldDef) => {
+				if (fieldDef?.column) {
+					return field === fieldDef.column;
+				}
 
-		return (
-			<>
-				<Label>{contentItem.label}:</Label>
-				<ChipsWrapper>
-					{contentItem.value.map((value, idx) => (
-						<Chip key={`${idx}-${value}`} label={value} />
-					))}
-				</ChipsWrapper>
-			</>
-		);
-	};
+				return field === fieldDef.name;
+			});
 
-	/**
-	 * Renders the content of type "file"
-	 * @param contentItem
-	 * @returns The info of the file passed within the label
-	 * and value attributes and it renders an icon if any.
-	 */
-	const fileContent = (contentItem: ContentType) => {
-		const FileIcon = contentItem?.icon;
-
-		return (
-			<>
-				<LabelWrapper>
-					{FileIcon && <FileIcon />}
-					<Label>{contentItem.label}:</Label>
-				</LabelWrapper>
-				<Value isFile>{contentItem.value}</Value>
-			</>
-		);
+			if (!currentField && field) {
+				throw new Error(
+					`No field declared for field name '${field}' in the ${column} column.`
+				);
+			}
+
+			if (currentField?.show && !showContent(currentField?.show)) {
+				return;
+			}
+
+			return currentField?.transforms.map((transform) => (
+				<FieldContainer key={currentField.name}>
+					<Label>{currentField.label}</Label>
+					{transform({ data: values[currentField?.column ? currentField?.column : currentField.name] })}
```

Comment:
We should wrap this in a div or span that passes the styles that are required in the figma (font-size: 14px, and color; gray600). I already tested this and it should only affect texts.